CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - Changed `SystemTable::firmware_revision` to return a `u32` instead of
   a `Revision`. The firmware revision's format is vendor specific and
   may not have the same semantics as the UEFI revision.
+- Changed `Revision` to `repr(transparent)`.
```

Comment:
I this a change or rather an addition? The only effect of this is that one can now `transmute` between `Revision` and `u32`, right?

Reply:
I think change is correct for `repr` changes in general, since it's changing the repr from the implicit Rust repr to something else.

In this specific case, it's also a correctness thing because the `Revision` struct is used in the headers of the system/boot/runtime tables. If Rust decided to lay out `Revision` as something other than 32 bytes with 4-byte alignment then the whole table could be messed up. And as far as I know, with the default repr, Rust would be free to do so.